src/libs/runtime-utils.js

```diff
@@ -280,7 +280,8 @@ export const trimRuntimesOldestFirst = _.flow(
 
 export const getCurrentRuntime = runtimes => {
   // Status note: undefined means still loading, null means no runtime
-  return !runtimes ? undefined : (_.flow(trimRuntimesOldestFirst, _.last)(runtimes) || null)
+  const runtime = !runtimes ? undefined : (_.flow(trimRuntimesOldestFirst, _.last)(runtimes) || null)
+  return runtime
```

Comment:
Is there a difference in functionality between the old code and new code?

Seems like we don't need to assign the runtime variable then return it. Should we revert this change?

Reply:
I wasn't super clear on what exactly the function was returning at first, and I did some further manipulation originally which I removed (though I guess the name makes the return pretty explicit). I can revert it though.

Reply:
+1 on reverting as the function name sounds descriptive to me